taccsite_cms/templates/guides/getting_started.html

```diff
@@ -0,0 +1,440 @@
+      </p>
+
+      <div class="row">
+        <div class="col">
+          <p>
+            <img
+              class="img-fluid"
+              src="https://utrc.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/cd/61/cd61c03b-1a46-4ab2-9446-fc5a0421d2c6/fig1-selectuniversity.png__525x409_q85_subsampling-2.png"
+            />
+          </p>
+        </div>
+
+        <div class="col">
+          <p>
+            First, you will need to create a TACC account using your UT System
+            Institution Credentials. Please navigate to the sign up page and
+            select your institution at
+            <a href="https://portal.tacc.utexas.edu/utdr" target="_blank" >portal.tacc.utexas.edu/utdr</a>
+            .
+          </p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col">
+          <p>
+            <img
+              class="img-fluid"
+              src="https://utrc.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/32/f7/32f786bc-56b4-4741-a7c4-987a1c5b3093/fig-2-eid.gif__525x197_q85_subsampling-2.png"
+            />
+          </p>
+        </div>
+
+        <div class="col">
+          <p>Log in with your institution credentials.</p>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col">
+          <p>
+            <img
+              class="img-fluid"
+              src="https://utrc.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/a5/44/a544ac7b-3b5d-431d-9a58-610ce0bcd4c1/fig-3-newaccount.png__525x201_q85_subsampling-2.png"
+            />
+          </p>
+        </div>
+
+        <div class="col">
+          <p>
+            You should see a page that welcomes you with your name and
+            institution. Click the
+            <strong>I need to create a TACC Account</strong> button. Complete
+            form. If you are eligible, check the
+            <strong>I am PI Eligible</strong> box. You will receive an e-mail
+            with a verification link. Please click on the link to verify your
+            account.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <h3>Logging In for the First Time</h3>
+
+      <p>
+        Before you can access the Portal, you must first have an active
+        account with TACC. You can create one that is linked to your UT System
+        Institution credentials.
+      </p>
+
+      <div class="row">
+        <div class="col">
+          <p>
+            <img
+              class="img-fluid"
+              src="https://utrc.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/e8/76/e87670ab-f330-4448-9414-f97549798225/login_page.png__525x249_q85_subsampling-2.png"
+            />
+          </p>
+        </div>
+
+        <div class="col">
+          <p>
+            You may now log in to the Portal by navigating to
+            <a href="https://utrc.tacc.utexas.edu" target="_blank">https://utrc.tacc.utexas.edu</a>
```

Comment:
Any way to turn this into the site url, perhaps with django `request.build_absolute_uri('/login')`

Reply:
Yes. I wish it was that easy. See 526c83057d58fc40c7cdfd58985734105fc7b058, then eca121e5cacfb1fb3a2fa3bb9d83919f3da7877d.

<details><summary>Long Story</summary>

> One cannot pass an argument to a function in a template tag without creating a custom template tag to allow that, but I don't think I want to allow such freedom in the templates. I found a basic template shortcut to get the desired output without creating a custom template tag (`{{ request.scheme }}://{{ request.get_host }}`), but this would be used more than once, which means I still need to build a custom template tag. So, I created one that provides a `site_uri` object.

</details>

I am open to suggestion(I have some) to my solution.

<details><summary>Other Ideas</summary>

- A new or replacement tag that just allows `request.build_absolute_uri(path_parameter_from_template)`.
- A new or replacement object that provides links to commonly-externally-referenced Portal paths.
    - Bonus: Rather than store the paths in the custom template tag, get them the portals `urls.py` (somehow).

</details>